pkgs/applications/kde/kgoldrunner.nix

```
@@ -0,0 +1,15 @@
+{ lib, mkDerivation, extra-cmake-modules, kio, kxmlgui, libkdegames }:
+
+mkDerivation {
+  pname = "kgoldrunner";
+
+  nativeBuildInputs = [ extra-cmake-modules ];
+  buildInputs = [ kio kxmlgui libkdegames ];
+
+  meta = with lib; {
+    homepage = "https://apps.kde.org/kgoldrunner/";
+    description = "Hunt gold, dodge enemies and solve puzzles";
+    license = licenses.gpl2Plus;
+    platforms = platforms.linux;
```

```suggestion
    platforms = platforms.linux;
    broken = versionOlder qtbase.version "5.13";
```
Like that @SuperSandro2000? Or is it just broken for 5.12?